modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -313,6 +318,36 @@ private synchronized void _generate(
 				"FDSView FDSDynamicFilter Relationship"),
 			"fdsViewFDSDynamicFilterRelationship",
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		ObjectDefinition fdsSortObjectDefinition =
+			_objectDefinitionLocalService.addCustomObjectDefinition(
+				userId, false, false,
+				LocalizedMapUtil.getLocalizedMap("FDS Sort"),
+				"FDSSort", "300", null,
+				LocalizedMapUtil.getLocalizedMap("FDS Sorts"),
+				ObjectDefinitionConstants.SCOPE_COMPANY,
+				ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT,
+				Arrays.asList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "field-name"), "fieldName", true),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "sorting"), "sorting", false)));
```

Comment:
I went with `sorting` since that's how the input is labeled in the UI. I've personally found naming variables to how they appear in the UI make it easier to figure out which variables affect the UI.

I could rename to maybe `sortingDirection`?

![Screenshot 2023-05-05 at 10 56 21 AM](https://user-images.githubusercontent.com/4496722/236532753-91873fc5-f2d5-442a-b474-da7c7a4e957e.png)

I'll make this field required. Thanks for the catch!

Reply:
Googling around, seems like "sort order" is how "ascending" and "descending" is typically described.
- "The default sort order is ascending" https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
- SQL has "ORDER BY"

So I'm thinking of naming this field to `sortingOrder`